gallery/src/styles/NxCard/NxCardPage.tsx

```diff
@@ -0,0 +1,152 @@
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card__call-out-text</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              In the case where the call out consists of text (e.g. 99%) it should be wrapped in this class (typically
+              in a <code className="nx-code">&lt;span&gt;</code>).
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card__text</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">Free-form text. Should be concise.</td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card__footer</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">Card footer. Typically provides a link to more information.</td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card--vertical-alt</code></td>
+            <td className="nx-cell">Modifier of <code className="nx-code">.nx-card</code></td>
+            <td className="nx-cell">
+              In order to use the variant vertical layout this class should be applied to
+              {' '}<code className="nx-code">.nx-card</code> in conjunction with a
+              {' '}<code className="nx-code">&lt;div&gt;</code> wrapping the call out and description text.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card__content</code></td>
+            <td className="nx-cell">Wrapping element</td>
+            <td className="nx-cell">
+              <code className="nx-code">.nx-card__content</code> applied to a
+              {' '}<code className="nx-code">&lt;div&gt;</code> wraps
+              {' '}<code className="nx-code">.nx-card__call-out</code> and
+              {' '}<code className="nx-code">.nx-card__call-out-text</code> in the alternate vertical card layout.
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </GalleryDescriptionTile>
+
+    <GalleryExampleTile title="Vertical Card"
+                        id="nx-card-vertical-example"
+                        liveExample={NxCardVerticalExample}
+                        codeExamples={nxCardVerticalCode}
+                        defaultCheckeredBackground={true}>
+      Cards shown in vertical orientation. While we have shown some content heavy examples it is recomended to keep the
+      content as minimal as possible. The orientation of the cards is determined by a class applied to their container.
+    </GalleryExampleTile>
+
+    <GalleryExampleTile title="Horizontal Card"
```

Comment:
I'm thinking it'd be a little nicer if the examples were reordered to put vertical and alt vertical next to each other.  I'm thinking horizontal first, then vertical, then alt vertical.

Reply:
> I'm thinking it'd be a little nicer if the examples were reordered to put vertical and alt vertical next to each other. I'm thinking horizontal first, then vertical, then alt vertical.

Sure: https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/ad954e98bdc482d6d24adf5542cd5265c659a9cd